os/windows/cpu-affinity.c

```diff
@@ -3,7 +3,7 @@
 #include <windows.h>
 
 /* Return all processors regardless of processor group */
```

Comment:
This is unneeded if we use the definition of `sysconf` from `os/windows/posix.c` although there uses `GetNumLogicalProcessors` whereas here uses `GetActiveProcessorCount`. I don't know much about Windows but it seems like `GetMaximumProcessorCount` is the equivalent to `_SC_NPROCESSORS_CONF`.

Reply:
I tested this on Windows and it turns out that using `GetMaximumProcessorCount` introduces a problem in `gettime.c` because Windows does not have `fio_get_thread_affinity`. So it seems like the safe thing to do is  to revert back to `GetActiveProcessorCount`.

```
$ ./fio.exe --name=test --ioengine=null --filesize=1M
fio: this platform does not support process shared mutexes, forcing use of threads. Use the 'thread' option to get rid of this warning.
test: (g=0): rw=read, bs=(R) 4096B-4096B, (W) 4096B-4096B, (T) 4096B-4096B, ioengine=null, iodepth=1
fio-3.12-1588-g93e8c
CPU mask contains processor beyond last active processor index (2)
clock setaffinity failed: No error
Starting 1 thread

test: (groupid=0, jobs=1): err= 0: pid=5900: Fri Oct 21 07:31:19 2022
  read: IOPS=256k, BW=1000MiB/s (1049MB/s)(1024KiB/1msec)
    clat (nsec): min=0, max=800, avg=74.61, stdev=64.01
     lat (nsec): min=100, max=1300, avg=135.55, stdev=86.47
    clat percentiles (nsec):
     |  1.00th=[    0],  5.00th=[    0], 10.00th=[    0], 20.00th=[    0],
     | 30.00th=[  100], 40.00th=[  100], 50.00th=[  100], 60.00th=[  100],
     | 70.00th=[  100], 80.00th=[  100], 90.00th=[  100], 95.00th=[  100],
     | 99.00th=[  100], 99.50th=[  100], 99.90th=[  804], 99.95th=[  804],
     | 99.99th=[  804]
  lat (nsec)   : 2=28.12%, 250=71.48%, 1000=0.39%
  cpu          : usr=0.00%, sys=0.00%, ctx=0, majf=0, minf=0
  IO depths    : 1=100.0%, 2=0.0%, 4=0.0%, 8=0.0%, 16=0.0%, 32=0.0%, >=64=0.0%
     submit    : 0=0.0%, 4=0.0%, 8=0.0%, 16=0.0%, 32=0.0%, 64=0.0%, >=64=0.0%
     complete  : 0=0.0%, 4=0.0%, 8=0.0%, 16=0.0%, 32=0.0%, 64=0.0%, >=64=0.0%
     issued rwts: total=256,0,0,0 short=0,0,0,0 dropped=0,0,0,0
     latency   : target=0, window=0, percentile=100.00%, depth=1

Run status group 0 (all jobs):
   READ: bw=1000MiB/s (1049MB/s), 1000MiB/s-1000MiB/s (1049MB/s-1049MB/s), io=1024KiB (1049kB), run=1-1msec
```

Reply:
I have replaced `GetMaximumProcessorCount` with `GetActiveProcessorCount` for this PR.